src/test/java/de/mediathekview/mserver/crawler/srf/parser/SrfSendungOverviewJsonDeserializerTest.java

```diff
@@ -20,52 +20,52 @@
+          new CrawlerUrlDTO("https://il.srgssr.ch/integrationlayer/2.0/srf/mediaComposition/video/99dbaab8-36e3-4f12-bf5f-34f197ed0777.json")
         }      
       },
       { "/srf/srf_sendung_overview_page_last.json", 
         Optional.empty(), 
-        new String[] { 
-          "https://il.srgssr.ch/integrationlayer/2.0/srf/mediaComposition/video/69cf918f-185a-4806-92f6-031e7f09844d.json&onlyChapters=true",
-          "https://il.srgssr.ch/integrationlayer/2.0/srf/mediaComposition/video/4eb1dbdf-dab8-4690-ba93-fdbafebbd5de.json&onlyChapters=true",
-          "https://il.srgssr.ch/integrationlayer/2.0/srf/mediaComposition/video/af4c9505-c265-49f6-86c8-67fe90dd0a2f.json&onlyChapters=true",
-          "https://il.srgssr.ch/integrationlayer/2.0/srf/mediaComposition/video/22b9dd2c-d1fd-463b-91de-d804eda74889.json&onlyChapters=true"
+        new CrawlerUrlDTO[] { 
+          new CrawlerUrlDTO("https://il.srgssr.ch/integrationlayer/2.0/srf/mediaComposition/video/69cf918f-185a-4806-92f6-031e7f09844d.json"),
+          new CrawlerUrlDTO("https://il.srgssr.ch/integrationlayer/2.0/srf/mediaComposition/video/4eb1dbdf-dab8-4690-ba93-fdbafebbd5de.json"),
+          new CrawlerUrlDTO("https://il.srgssr.ch/integrationlayer/2.0/srf/mediaComposition/video/af4c9505-c265-49f6-86c8-67fe90dd0a2f.json"),
+          new CrawlerUrlDTO("https://il.srgssr.ch/integrationlayer/2.0/srf/mediaComposition/video/22b9dd2c-d1fd-463b-91de-d804eda74889.json")
         } 
       },
     });
   }
   
   private final String jsonFile;
   private final Optional<String> expectedNextPageId;
-  private final String[] expectedUrls;
+  private final CrawlerUrlDTO[] expectedUrls;
   private final SrfSendungOverviewJsonDeserializer target;
   
-  public SrfSendungOverviewJsonDeserializerTest(String aJsonFile, Optional<String> aExpectedNextPageId, String[] aExpectedUrls) {
+  public SrfSendungOverviewJsonDeserializerTest(String aJsonFile, Optional<String> aExpectedNextPageId, CrawlerUrlDTO[] aExpectedUrls) {
     jsonFile = aJsonFile;
     expectedNextPageId = aExpectedNextPageId;
     expectedUrls = aExpectedUrls;
     
-    target = new SrfSendungOverviewJsonDeserializer();
+    target = new SrfSendungOverviewJsonDeserializer("https://www.srf.ch");
   }
   
   @Test
   public void test() {
     JsonElement jsonElement = JsonFileReader.readJson(jsonFile);
     
-    SrfSendungOverviewDTO actual = target.deserialize(jsonElement, SrfSendungOverviewDTO.class, null);
+    Optional<SrfSendungOverviewDTO> actual = target.deserialize(jsonElement, SrfSendungOverviewDTO.class, null);
     
-    assertThat(actual, notNullValue());
-    assertThat(actual.getNextPageId(), equalTo(expectedNextPageId));
-    assertThat(actual.getUrls(), Matchers.containsInAnyOrder(expectedUrls));
+    assertThat(actual.isPresent(), equalTo(true));
+    assertThat(actual.get().getNextPageId(), equalTo(expectedNextPageId));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "actual.isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3655)